libyui/libyui-ncurses
# Fix Menu Hotkeys in NCurses

## Bugzilla

https://bugzilla.suse.com/show_bug.cgi?id=1177760

## Problem

The hotkeys in NCurses menus were gone: Both in the pulldown menus of a MenuBar and in the pop-up menus of a MenuButton.


## Fix

I bisected the recent commits on NCurses and found the offending commits. I broke down the more recent one even more and found it was just that one function that broke it. Reverting that function to the previous really simplistic content fixed the problem.

We'll still have to check what exactly the underlying problem was; right now it's just breaking down commits and experimenting what helps and what does not.

## How to Test

Use the UI examples:

```
y2base ./MenuBar1.rb ncurses
y2base ./MenuButton1.rb ncurses
```

Also use the NCurses package selector:

```
sudo yast sw_single
```

Open the menus and check if the hotkeys are highlighted and do something.
